bookmarks/models.py

```diff
@@ -38,9 +38,10 @@ def get_attribution(self):
         users = User.objects.filter(sounds__in=sounds_list).distinct()
         # Generate text file with license info
         licenses = License.objects.filter(sound__id__in = sounds_list).distinct()
-        attribution = render_to_string("bookmarks/bookmark_category_attribution.txt",
-            dict(users=users,
-                category=self,
+        attribution = render_to_string(("sounds/multiple_sounds_attribution.txt"),
+            dict(type=self.__class__.__name__,
```

Comment:
`type="bookmark_category"` (or whatever you use to compare in the template). It will be clearer for our future selves this way.